### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ananya88**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,21 @@ Folly (acronymed loosely after Facebook Open Source Library) is a
 library of C++14 components designed with practicality and efficiency
 in mind. **Folly contains a variety of core library components used extensively
 at Facebook**. In particular, it's often a dependency of Facebook's other
-open source C++ efforts and place where those projects can share code.
+open source C++ efforts and places where those projects code can be shared.
```
Comment (yfeldblum) [inline README.md:12]: All of the changes on this line are incorrect.

File: README.md [hunk 2]
```diff
@@ -9,15 +9,21 @@ Folly (acronymed loosely after Facebook Open Source Library) is a
...
 
 It complements (as opposed to competing against) offerings
 such as Boost and of course `std`. In fact, we embark on defining our
 own component only when something we need is either not available, or
 does not meet the needed performance profile. We endeavor to remove
 things from folly if or when `std` or Boost obsoletes them.
 
-Performance concerns permeate much of Folly, sometimes leading to
+Folly is fast - The utilities contained in Folly are things we use
```
Comment (yfeldblum) [inline README.md:20]: Not sure that this paragraph is necessary.

Comment (Ananya88) [reply README.md:20]: okay, thanks for your response

**Comment by ahornby** [comment]
README is updated

**Pull request closed without merging**